src/routes/inventory-books.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file: routes/inventory-books.js
+ * @author dafquers
+ * @version 0.0.1
+ *
+ * This module add books to inventory.
+ */
+
+const db = require("../db/mongoConnector");
+const express = require("express");
+const books = require("./template-books");
+const { route } = require("./public");
+
+const router = express.Router();
+
+/**
+ * Search books.
+ *
+ * @path /insert_inventory_books
+ * @method POST
+ */
+
+router.route(["/insert_inventory_books"]).get((req,res)=>{
+  const user_param = req.session.username;
+  if (user_param == null) {
+    res.redirect("/login");
+  }else{
+    let inventory_name = req.query.title;
+    res.render("template/books-template.html",{inventory_name : inventory_name,user_param});
+  }
+})
+
+/**
+ * Insert inventory of type books.
+ *
+ * @path /add_books
+ * @method POST
+ */
+
+router.route(["/add_books"]).post(async (req,res)=>{
+  const user_param = req.session.username;
+  if (user_param == null) {
+    res.redirect("/login");
+  } else {
+    //an obj of the item to insert
+    let title = req.query.title;
+    let description = req.query.description;
+    let author = req.query.author;
+
+    let data = {
+      coll: req.query.inv_name,
+      items: {Titre:title,Auteur:author,Description:description},
+      email: req.session.email.replace(".", "_"),
+    };
+
+    await db.insertItemInventory(data);
+    res.redirect(`/show_inventory?title=${data.coll}`);
+  }
+})
+
+/**
+ * Create of inventory of type books.
+ *
+ * @path /add_books
+ * @method Get
```

Comment:
Please be consistent: `GET`
```suggestion
 * @method GET
```